core.py

```diff
@@ -472,6 +454,19 @@ def _setHidden(self, val):
 		else:
 			self.element.removeAttribute("hidden")
 
+	def _getDisabled(self):
+		return bool(self._disabledState)
+
+	def _setDisabled(self, disable):
+		for child in self._children:
+			child._setDisabled(disable)
+
+		if self._disabledState == 0:
+			if isinstance(self, _attrDisabled):
+				self.element.disabled = not disable
```

Comment:
Shall this really be `not`?

Reply:
@sveneberth you're totally right! I've recently updated the pull requests to a more convenient and error-reistant version. 
In the original version, self._disabledState could also become negative, which is not the desired behavior.